extensions/dicom-microscopy/src/components/ViewportOverlay/index.tsx

```diff
@@ -88,7 +88,7 @@ export const generateFromConfig = ({ config, overlayData, ...props }) => {
 
 const itemGenerator = (props: any) => {
   const { item } = props;
-  const { title, value: valueFunc, condition, contents } = item;
+  const { title, value: valueFunc, contentF: contentFunc, condition, contents } = item;
```

Comment:
const { title, value: valueFunc, contentF: contentFunc = valueFunc, condition, contents } = item;
will assign to contentFunc directly